watson/cli.py

```diff
@@ -14,39 +14,8 @@
 
 from . import watson
 from .frames import Frame
-from .utils import format_timedelta, sorted_groupby, options
-
-
-def style(name, element):
-    def _style_tags(tags):
-        if not tags:
-            return ''
-
-        return '[{}]'.format(', '.join(
-            style('tag', tag) for tag in tags
-        ))
-
-    def _style_short_id(id):
-        return style('id', id[:7])
-
-    formats = {
-        'project': {'fg': 'magenta'},
-        'tags': _style_tags,
-        'tag': {'fg': 'blue'},
-        'time': {'fg': 'green'},
-        'error': {'fg': 'red'},
-        'date': {'fg': 'cyan'},
-        'short_id': _style_short_id,
-        'id': {'fg': 'white'}
-    }
-
-    fmt = formats.get(name, {})
-
-    if isinstance(fmt, dict):
-        return click.style(element, **fmt)
-    else:
-        # The fmt might be a function if we need to do some computation
-        return fmt(element)
+from .utils import style, format_timedelta, sorted_groupby, options
+from .utils import get_frame_from_argument
```

Comment:
Can you merge these imports with parenthesis notation and sort them alphabetically please :)